src/components/SensorOverlay/components/DraggableBox.tsx

```diff
@@ -266,20 +286,17 @@ export class DraggableBox
   };
 
   fetchTimeSeries = async (id: number) => {
+    if (!this.context) {
```

Comment:
It would be nice to have some service messages that will notify user to wrap components in `ClientSDKProvider`